src/features/navigation/RootNavigator/linking/getInitialUrl.ios.ts

```diff
@@ -11,7 +11,7 @@ export async function getInitialURL(): Promise<string> {
   if (dynamicLinkUrl) {
     return dynamicLinkUrl.url
   }
-  const url = await Linking.getInitialURL()
+  const url = await BatchPush.getInitialURL()
```

Comment:
Est-ce que c'est une surcharge ou complémentaire ?

Reply:
(On peut avoir des UniversalLink qui ne viennent pas de push notifs)

Reply:
C'est une surcharge: cf la doc: https://doc.batch.com/react-native/advanced/deeplinking#doc_page_title

Reply:
> This is a replacement of Linking.getInitialURL that you can use on Android or iOS: Batch.getInitialURL first checks if Linking.getInitialURL returns something, and then if it doesn't on iOS, it calls a custom native function of this module that gets the first deeplink it has ever seen. Subsequent calls to this native function will return null to prevent a future JS reload to see an old initial URL. Because of this, you have to make sure to call this method only once in the app lifetime.